pgctl/fuser.py

```diff
@@ -22,7 +26,7 @@ def stat(path):
         trace('fuser suppressed: %s', error)
         return None
     else:
-        return (path.st_ino, path.st_dev)
+        return StatTuple(path.st_ino, path.st_dev, path.st_nlink)
```

Comment:
I needed to add st_nlink so I can check it's 0. I chose to make it a NamedTuple instead of a plain tuple to be more readable.